servers/src/test/java/tachyon/worker/block/evictor/EvictorTest.java

```diff
@@ -87,7 +91,9 @@ public final void before() throws Exception {
             Collections.<Long>emptySet());
     List<StorageTier> tiers = mMetaManager.getTiers();
     mTestDir = tiers.get(TEST_TIER_LEVEL).getDir(TEST_DIR);
-    mEvictor = EvictorFactory.create(mEvictorType, mManagerView);
+    Map<String, String> props = new LinkedHashMap<String, String>();
+    props.put(Constants.WORKER_EVICT_STRATEGY_TYPE, mEvictorClassName);
```

Comment:
You can create a TachyonConf(), and then set() the evictor class name, instead of filling the Map directly.